Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -802,8 +802,11 @@ void function FilterServerList()
 	}
 	
 	// Update player and server count
-	Hud_SetText( Hud_GetChild( file.menu, "InGamePlayerLabel" ), Localize("#INGAME_PLAYERS", string( totalPlayers ) ) )
-	Hud_SetText( Hud_GetChild( file.menu, "TotalServerLabel" ),  Localize("#TOTAL_SERVERS", string( NSGetServerCount() ) ) )
+	int ServerCount = NSGetServerCount()
+	string totalPlayersStr = string( totalPlayers ) + ( totalPlayers == 1 ? " " : ""  ) + ( string( totalPlayers ).len() == 1 ? " " : ""  )
+	string serverCountStr = string( ServerCount ) + ( ServerCount == 1 ? " " : "" ) + ( string( ServerCount ).len() == 1 ? " " : ""  )
```

Comment:
why are you adding a space twice here? ( well only when servercount or totalplayers would be equal to 1 )
why not just have the check for the amount of digits? 
also using strings to get the amount of digits is funny.